python/ct/proto/client.proto

```diff
@@ -205,3 +205,9 @@ message SignedCertificateTimestamp {
                                   (tls_opts).max_length = 0xffff ];
   optional DigitallySigned signature = 5;
 }
+
+message SignedCertificateTimestampList {
+  repeated bytes sct_list = 1 [ (tls_opts).min_length = 1,
```

Comment:
Could test encode/decode failure on empty SCTs.

OTOH the advantage of the generic encoder is that  you shouldn't have to repeat those tests at the application layer, so up to you.